src/MUIDataTable.js

```diff
@@ -80,6 +80,7 @@ class MUIDataTable extends React.Component {
             viewColumns: PropTypes.bool,
             filterList: PropTypes.array,
             filterOptions: PropTypes.array,
+            filterType: PropTypes.oneOf(['dropdown', 'checkbox', 'multiselect', 'textField']),
```

Comment:
The filters will not filter properly unless you change line 481 of this file with this (or something similar)
```
      const { caseSensitive } = this.options;
      let { filterType } = this.options;
      if (columns[index].options && columns[index].options.filterType) {
        filterType = columns[index].options.filterType;
      }
```
I tried with a `filterType: "textField"` on the age to test this out 😉

Reply:
Fixed it.